Makefile

```diff
@@ -26,4 +27,22 @@ vet:
 # Compile and install sbctl locally in you GOBIN path
 .PHONY: install
 install:
-	go install sbctl.go
+	go install ${BUILDFLAGS} sbctl.go
+
+.PHONY: lint
+lint:
+	ifeq (, $(shell which golangci-lint))
+		$(error "Install golangci-lint by either running 'make install-golangci-lint' or by other means")
+	endif
+	golangci-lint run --new -c .golangci.yaml --build-tags ${BUILDTAGS} ${BUILDPATHS}
+
+.PHONY: lint-and-fix
+lint-and-fix:
+	ifeq (, $(shell which golangci-lint))
+		$(error "Install golangci-lint by either running 'make install-golangci-lint' or by other means")
+	endif
+	golangci-lint run --new --fix -c .golangci.yaml --build-tags ${BUILDTAGS} ${BUILDPATHS}
+
+.PHONY: install-golangci-lint
+install-golangci-lint:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest
```

Comment:
do we really need to add this? asking since we're not versioning it in here.. 

and while comparing, I didn't see one for [troubleshoot](https://github.com/replicatedhq/troubleshoot/blob/main/Makefile#L235-L241) (actually, why isn't there a lint checker in CI there?)

Reply:
> and while comparing, I didn't see one for [troubleshoot](https://github.com/replicatedhq/troubleshoot/blob/main/Makefile#L235-L241) (actually, why isn't there a lint checker in CI there?)

There is an issue in the backlog for this, but its not been worked on full. There are [`make` targets](https://github.com/replicatedhq/troubleshoot/blob/e3adc1cb35d8e3929268b4b47879603128608de8/Makefile#L235-L241) in the codebase but no CI integration yet

https://github.com/replicatedhq/troubleshoot/issues/754

Reply:
> do we really need to add this? asking since we're not versioning it in here..

The make target is used to install golangci-lint in the GH action running tests. We use the latest to avoid having to manually track versioning the tool. We use very few linters to expect any side effects of tool updates. Usually these would be linters getting upgraded or new linters getting introduced.

I added a [comment](https://github.com/replicatedhq/sbctl/pull/124/commits/dfa2cce00a1f8dbe80722b34be37b392e3cf8bef) on the same.